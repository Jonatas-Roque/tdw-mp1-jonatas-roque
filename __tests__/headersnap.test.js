import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/header';

test('Header snapshot', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});