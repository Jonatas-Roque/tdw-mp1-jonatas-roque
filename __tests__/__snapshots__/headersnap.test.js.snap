// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
<h2
  className="text-2xl md:text-4xl font-bold tracking-tight md:tracking-tighter leading-tight mb-20 mt-8"
>
  <a
    className="hover:underline"
    href="/"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onTouchStart={[Function]}
  >
    Blog do JÃ³natas
  </a>
  .
</h2>
`;
